packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -25,10 +25,14 @@ import KeyDidResolver from 'key-did-resolver'
 import { Resolver } from 'did-resolver'
 import { DID } from 'dids'
 
+const DEFAULT_DAEMON_CONFIG_FILE = 'daemon.config.json'
 const DEFAULT_CLI_CONFIG_FILE = 'client.config.json'
-const LEGACY_CLI_CONFIG_FILE = 'config.json' // todo(1615): Remove this backwards compatibily support
-const DEFAULT_CLI_CONFIG_PATH = path.join(os.homedir(), '.ceramic')
-const DEFAULT_NETWORK = Networks.TESTNET_CLAY
+const LEGACY_CLI_CONFIG_FILE = 'config.json' // todo(1615): Remove this backwards compatibility support
+const DEFAULT_CONFIG_PATH = path.join(os.homedir(), '.ceramic')
+
+const DEFAULT_DAEMON_CONFIG: DaemonConfig = {
+  network: Networks.TESTNET_CLAY,
+}
```

Comment:
Is this the only thing that is needed in the default config?

Reply:
I added https://github.com/ceramicnetwork/js-ceramic/issues/1632 to revisit what the defaults should be